main.go

```diff
@@ -132,77 +136,73 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
-	milestones := []milestoneAPI{}
-
-	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
-	URL := strings.Join(strURL, "")
-
-	m, linkHeader, err := callServer(URL, token, milestones)
-	if err != nil {
-		return m, err
-	}
+func getActiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "active"
+	return getMilestones(baseURL, token, projectID, state)
+}
 
-	for _, milestone := range milestones {
-		m[milestone.Title] = milestone.DueDate
-	}
+// Get and return inactive milestones
+func getInactiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "closed"
+	return getMilestones(baseURL, token, projectID, state)
+}
 
-	parsedHeader := link.Parse(linkHeader)
-	for _, elem := range parsedHeader {
-		if elem.Rel == "next" {
-			m, _, err := callServer(URL, token, milestones)
-			if err != nil {
-				return m, err
+func getMilestones(baseURL string, token string, projectID string, state string) ([]milestoneAPI, error) {
+	m := []milestoneAPI{}
+	client := &http.Client{}
+	strURL := []string{baseURL, "/projects/", projectID, "/milestones"}
+	URL := strings.Join(strURL, "")
+	params := url.Values{}
+	params.Add("state", state)
+	for {
+		next := false
```

Comment:
don't set false here or we finish the loop early. we will set that instead of the break